examples/inverse/plot_morph_VolSourceEstimate.py

```diff
@@ -0,0 +1,282 @@
+"""
+======================================
+Demonstrate morphing VolSourceEstimate
+======================================
+
+This example demonstrates how to morph an individual subject source space to a
+common reference space. For this purpose `dipy
+<http://nipy.org/dipy>`_ will be
+used in order to perform the necessary transforms to a fsaverage serving as
+reference space. The example uses parts of the MNE example
+:ref:`LCMV beamformer pipeline
+<sphx_glr_auto_examples_inverse_plot_lcmv_beamformer_volume.py>`.
+The respective result will be morphed based on an affine transformation and a
+nonlinear morph, estimated based on respective transformation from the
+subject's anatomical T1 (brain) to fsaverage T1 (brain). Afterwards the
+transformation will be applied to the beamformer result. Affine transformations
+are computed based on the mutual information. This metric relates structural
+changes in image intensity values. Because different still brains expose high
+structural similarities this method works quite well to relate corresponding
+features [1]_. The nonlinear transformations will be performed as
+Symmetric Diffeomorphic Registration using the cross-correlation metric [2]_.
+
+.. note:: This example applies downsampling to all volumes in order to speed up
+        computation. In a real case scenario you might want to reconsider if
+        and to what extend resliceing might be necessary.
+
+References
+----------
+
+.. [1] Mattes, D., Haynor, D. R., Vesselle, H., Lewellen, T. K., &
+        Eubank, W. (2003). PET-CT image registration in the chest using
+        free-form deformations. IEEE transactions on medical imaging, 22(1),
+        120-128.
+
+.. [2] Avants, B. B., Epstein, C. L., Grossman, M., & Gee, J. C. (2009).
+        Symmetric Diffeomorphic Image Registration with Cross- Correlation:
+        Evaluating Automated Labeling of Elderly and Neurodegenerative Brain,
+        12(1), 26-41.
+
+"""
+# Author: Tommy Clausner <tommy.clausner@gmail.com>
+#
+# License: BSD (3-clause)
+
+from os import makedirs, path
+
+import matplotlib.pylab as plt
+import mne
+import nibabel as nib
+import numpy as np
+from dipy.align import imaffine, imwarp, reslice
+from mne.beamformer import make_lcmv, apply_lcmv
+from mne.datasets import sample
+from mne.externals.h5io import write_hdf5, read_hdf5
+from nilearn.image import index_img
+from nilearn.plotting import plot_anat
```

Comment:
please fix ordering of imports.

it should go from more general to specific. So here mne should be last
std lib -> numpy -> nibabel/nilearn -> mne